pyproject.toml

```diff
@@ -32,7 +32,7 @@ web3 = "<7,>=6.0.0"
 ipfshttpclient = "==0.8.0a2"
 open-aea-cli-ipfs = "==1.52.0"
 aiohttp = "<4.0.0,>=3.8.5"
-certifi = "*"
+certifi = "==2021.10.8"
```

Comment:
Any idea why this has changed?

Reply:
I think you ran `tox -e check-dependencies`.